addon/authenticators/jwt.js

```diff
@@ -120,11 +122,11 @@ export default TokenAuthenticator.extend({
 
```

Comment:
From what I am seeing, if the authentication is expired when it enters the restore function then it will reject on line 132
```
 reject(new Error('token is expired'));
```